include/cpp-statsd-client/StatsdClient.hpp

```diff
@@ -63,7 +65,8 @@ class StatsdClient {
                  const uint16_t port,
                  const std::string& prefix,
                  const uint64_t batchsize = 0,
-                 const uint64_t sendInterval = 1000) noexcept;
+                 const uint64_t sendInterval = 1000,
+                 const unsigned int gaugePrecision = 4) noexcept;
```

Comment:
We could also add a quick note about the setting float precision (a line or two is enough) in the README, like we did for the other parameters.